src/style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.prod {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+}
+
+.prod__img {
+  width: 100%;
+  text-align: center;
+  height: 134px;
+  padding: 0 3px;
+}
+
+.prod__img > img {
+  width: 160px;
+  height: 134px;
+}
+
+.prod__about {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-top: 40px;
+}
+
+.prod__id {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.stars-row {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(./images/star.svg) no-repeat center;
+}
+
+.stars-row > .star:nth-child(n+5) {
+  margin: 0;
+}
+
+.stars-row > .star:nth-child(-n+4) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
```

Comment:
You should also add styles for cases with 1, 2, 3, 5 active stars, and use modifier to manage it